endpoints/books/controller.py

```diff
@@ -4,8 +4,12 @@
 from app import db
 from app import api
 from utilities import responseSchema
+import time
+from flask_jwt_extended import ( jwt_required, jwt_refresh_token_required, get_jwt_identity)
+from .repository import UserBookRepository
+# response = responseSchema.ResponseSchema()
+userBookRepository = UserBookRepository()
```

Comment:
Try using dependency injection or to pass it per request instead of initializing it here